components/form/input/src/Password/index.scss

```diff
@@ -0,0 +1,18 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$r-form-input-password-toggle-button: $gutter !default;
+$pr-form-input-password-input: 60px !default;
```

Comment:
This should be a padding or margin variable.

Reply:
I'll remove it as it's not used